pkg/gofr/datasource/pubsub/eventhub/eventhub.go

```diff
@@ -21,7 +20,10 @@ import (
 // metrics are being registered in the container, and we are using the same metrics so we have not re-registered the metrics here.
 // It is different from other datasources.
 
-var errNoMsgReceived = errors.New("no message received")
+var (
+	errNoMsgReceived = errors.New("no message received")
+	errTopicMismatch = errors.New("topic should be same as eventhub name")
```

Comment:
Here and everywhere

The name is `Event Hub` with this exact case.


```suggestion
	errTopicMismatch = errors.New("topic should be same as Event Hub name")
```

Reply:
We can have a constant and use it everywhere would make it consistent

Reply:
At least a forbiddigo rule looking for well known names should be added I think

Reply:
Tried making a constant for this but then for every error message we needed to use `fmt.Sprintf` to use that constant which doesn't looked very good. So changed the name to `Event Hub` everywhere.

Reply:
i wouldn't have recommended constants, yes.

That's why I suggested using forbidigo